calculator/main.py

```diff
@@ -58,18 +78,24 @@ def get_history(calc):
         ie:
         ('2016-05-20 12:00:00', 'add', (1, 2), 3),
     """
-    pass
+    return calc['history']
 
 
 def reset_history(calc):
     """
     Resets the calculator history back to an empty list.
     """
-    pass
-
+    del(calc['history'])
+    calc.update({'history': []})
+    
 
 def repeat_last_operation(calc):
     """
     Returns the result of the last operation executed in the history.
     """
-    pass
+    if calc['history'] != []:
+        return calc['history'][-1][3]               
+    else:
+        return None
```

Comment:
Can you guys think in a one line solution for this? (just for learning purposes, so you smash your head against the wall for a few minutes)